### ISSUE TITLE
Hide query loop pagination link arrows from assistive technology

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#41940]
### Description

The core ticket https://core.trac.wordpress.org/ticket/56067 proposes to hide from assistive technology the arrow or chevron characters used for the next/previous pagination links. It applies to the Query Loop > Pagination and to the Comments Query Loop > Comments Pagination blocks.

The core ticket takes care of the front end. This issue aims to evaluate whether an `aria-hidden="true'` attribute should be added also for:

- The backwards compatibility functions for older versions of WordPress, see in the files `lib/compat/wordpress-5.9/blocks.php` and `lib/compat/wordpress-6.0/blocks.php`.
- More importantly: the edit components of the next / previous links.

While the edit components relate to a different context (editing), I'd tend to think it's best to make sure the user experience is the same in the editor and in the front end. Adding an `aria-hidden="true"` attribute seems the best option.

### Step-by-step reproduction instructions

- Use Safari and VoiceOver (or other combo)
- Make sure there are more than 10 published posts
- Go to the site Editor with the Twenty Twenty-Two theme activated
- Navigate the Home template by using VoiceOver and go to the 'Next Page' block in the pagination
- Observe VoiceOver reads out the arrow as 'right arrow'.

### Please confirm that you have searched existing issues in the repo.

Yes

### Please confirm that you have tested with all plugins deactivated except Gutenberg.

Yes
